site.yml

```diff
@@ -14,6 +14,18 @@
   - { role: base }
   - { role: serf }
   - { role: dev }
+  - { role: vagrant }
+
+- hosts: build
```

Comment:
cool, thanks @erikh . I have a patch (https://github.com/mapuri/ansible/commit/0f1d8da5e4ae922fdb4b23f1b235e347edf69867) that I am trying out with your virtualbox guest-additions fix.

I will send a PR in ansible and build once I have it tested out a bit.